### DISCUSSION TITLE
CLI "chia plotnft inspect"  return "state=3",  Does this mean mining?

### DISCUSSION THREAD

**Discussion by Luuuxury** [Chia-Network/chia-blockchain#7726] [Support]
`(venv) root@lux:~/chia-blockchain# chia plotnft inspect -i 3
{'pool_wallet_info': PoolWalletInfo(current=PoolState(version=1, state=3, target_puzzle_hash=<bytes32: 2f2c9ba1b2315d413a92b5f034fa03282ccba1767fd9ae7b14d942b...... bala bala ..........`

The thing that confuses me is how do I know I'm already mining in a pool? 
Does a status of 3 mean there is no problem? I mean I'm mining now?

**Comment by emlowe** [comment]
chia plotnft show 
would tell you farming to pool

which is what state==3 means

You can check with your pool to make sure they are getting partials
